build-doc/ansible_module_documentation.py

```diff
@@ -112,6 +121,8 @@ def main():
                 doc, plainexamples, returndocs = module_docs.get_docstring(module_file_name)
```

Comment:
I made the following change and the documentation was generated using Ansible v2.3:

```python
doc, plainexamples, returndocs, _ = module_docs.get_docstring(module_file_name)
````